tests/correctness/ref.expect

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
You have both `ref.expect` and `ref.skip` in the directory which means `runt` will not run this test. I guess the interpreter uses the same test path as the compiler. One possible solution is creating a `tests/correctness/ref-cells/` directory and placing the `ref cell` tests there. Then we can change the `runt` definition for the compiler to test those files. The interpreter correctness tests will not use that path and therefore not cause any failures.

Reply:
To do this do we just modify the runt.toml file?

Reply:
You also need to move the test to a new directory because by default, the “*.futil” glob in the test paths will try to run everything matching it